src/main/java/io/sealights/plugins/sealightsjenkins/BeginAnalysis.java

```diff
@@ -654,6 +658,24 @@ private SeaLightsPluginInfo createSeaLightsPluginInfo(
         slInfo.setBuildFilesFolders(foldersToSearch);
         slInfo.setBuildFilesPatterns(patternsToSearch);
 
+        if(additionalProps.get(SL_JVM_PARAMS_KEY)!= null){
+            String jvmParams = (String)additionalProps.get(SL_JVM_PARAMS_KEY);
+            Map<String, String> paramsMap = StringUtils.convertKeyValueStringToMap(jvmParams);
```

Comment:
Can you create a function that checks for null and returns a map? Seems like a lot of code duplication.